src/nft/queries/genie/TrendingTokensFetcher.ts

```diff
@@ -1,6 +1,9 @@
+import { unwrapToken } from 'graphql/data/util'
+
 import { FungibleToken } from '../../types'
 
 export const fetchTrendingTokens = async (numTokens?: number): Promise<FungibleToken[]> => {
+  // TODO: WETH->ETH
```

Comment:
this todo is not very descriptive tbh

Reply:
That was a personal TODO so I could easily find the relevant codepaths :)